src/widgets/Notebook.cpp

```diff
@@ -325,6 +325,48 @@ void Notebook::setAllowUserTabManagement(bool value)
     this->allowUserTabManagement_ = value;
 }
 
+bool Notebook::getShowTabs() const
+{
+    return this->showTabs_;
+}
+
+void Notebook::setShowTabs(bool value)
+{
+    this->showTabs_ = value;
+
+    this->performLayout();
+    for (auto &item : this->items_)
+    {
+        item.tab->setHidden(!value);
+    }
+
+    this->setShowAddButton(value);
+
+    // show a popup upon hiding tabs
+    if (!value && getSettings()->informOnTabVisibilityToggle.getValue())
+    {
+        QMessageBox msgBox;
+        msgBox.window()->setWindowTitle("Chatterino - hidden tabs");
+        msgBox.setText("Hide tabs");
+        msgBox.setInformativeText(
+            "You've just hidden your tabs.\nYou can (un)hide those with "
+            "either keyboard shortcut (Default: Ctrl+U) or select an option "
+            "in split header's menu.");
```

Comment:
This wording feels a bit odd - especially if you're a new chatterino user. I suggest the following:

```
You've just hidden your tabs!
You can toggle tabs by using the keyboard shortcut (Ctrl+U by default) or clicking "toggle visibility of tabs" in the dropdown next to the stream title.
```

Reply:
Thank you very much for feedback from a native english speaker :heart: